rules/opentitan/fpga_cw310.bzl

```diff
@@ -341,24 +317,8 @@ def hyper310_jtag_params(
     Returns:
       struct of test parameters.
     """
-    if bitstream and (rom or otp):
-        fail("Cannot use rom or otp with bitstream.")
-    if not bitstream:
-        bitstream = "@//hw/bitstream/universal:splice"
-    return struct(
-        tags = ["hyper310", "exclusive"] + tags,
```

Comment:
The interface-specific tags have been dropped here. We seem to have moved over to using a tag derived from the Target's label, though, so maybe we can just eliminate the interface tag for cw310 as well.

We'll probably need to overhaul test filtering for multi-top.

Reply:
Good point. I think the current "hyper310", "cw310" label isn't accurate though because there's no `hyper310_params` so anything not using jtag is still tagged with `cw310`. I'll see if any of our CI relies on the interface tag.

Reply:
That's a good point but I agree with @nbdd0121 that those labels are not accurate, I think the exec_env labels are more reliable.